packages/react-devtools-shared/src/__tests__/__snapshots__/store-test.js.snap

```diff
@@ -1,5 +1,63 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Store Lazy should support Lazy components (legacy render): 1: mounted + loading 1`] = `
+[root]
+  ▾ <App>
+      <Suspense>
+`;
+
+exports[`Store Lazy should support Lazy components (legacy render): 2: mounted + loaded 1`] = `
+[root]
+  ▾ <App>
+    ▾ <Suspense>
+        <LazyInnerComponent>
+`;
+
+exports[`Store Lazy should support Lazy components (legacy render): 3: unmounted 1`] = `
+[root]
+    <App>
+`;
+
+exports[`Store Lazy should support Lazy components in (createRoot): 1: mounted + loading 1`] = `
+[root]
+  ▾ <App>
+      <Suspense>
+`;
+
+exports[`Store Lazy should support Lazy components in (createRoot): 2: mounted + loaded 1`] = `
```

Comment:
Nit: ideally these descriptions could describe what's expected in the snapshots so we can easy review whether the test is correct or not.